sunpy/visualization/animator/tests/test_basefuncanimator.py

```diff
@@ -206,5 +211,8 @@ def test_imageanimator_figure():
     time_diff = (t1 - t0).to(u.s)
     wcs_input_dict.update({'CTYPE1': 'Time', 'CUNIT1': time_diff.unit.name, 'CDELT1': time_diff.value})
     wcs = astropy.wcs.WCS(wcs_input_dict)
-    wcs_anim = ImageAnimatorWCS(sequence_array, wcs=wcs, vmax=1000, image_axes=[0, 1])
-    plt.show()
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        wcs_anim = ImageAnimatorWCS(sequence_array, wcs=wcs, vmax=1000, image_axes=[0, 1])
```

Comment:
What warnings are being thrown up, and is it possible to do a more specific warning filter instead of just ignoring everything?

Reply:
it was to filter the `SunPyDeprecationWarning` I will change it :+1: